### ISSUE TITLE
Automatically fixing tag-lines reformats the description, as well as removing tag lines

### ISSUE THREAD

**Issue by Standard8** [gajus/eslint-plugin-jsdoc#1593]
## Expected behavior

When I automatically fix tag-lines issues, only the additional tag lines should be removed. The rest of the jsdoc should remain unchanged.

## Actual behavior

Automatic fixing additionally reformats the rest of the comment. For the test case below, the diff result is:

```js
diff --git a/test.js b/test.js
index b517e22..6e1e2d6 100644
--- a/test.js
+++ b/test.js
@@ -1,13 +1,12 @@
 /**
  * My Test Function, with some example code.
  *
- *   - Indented paragraph.
+ * - Indented paragraph.
  *
  * ```js
- *   new Foo();
+ * new Foo();
  * ```
  *
- *
  * @param {string} bar
  */
 function myTestFunction(bar) {
```

The indentation of the code sample and the `- Indented paragraph.` lines should have remained unchanged.

## ESLint Config

```js
import jsdoc from "eslint-plugin-jsdoc";

export default [{
  plugins: {jsdoc},
  rules: {
    "jsdoc/tag-lines": ["error", "any", { startLines: 1 }],
  }
}]
```

## ESLint sample

```js
/**
 * My Test Function, with some example code.
 *
 * ```js
 *   new Foo();
 * ```
 *
 *
 * @param {string} bar
 */
function myTestFunction(bar) {

}
```

## Environment

- Node version: v25.1.0
- ESLint version v9.39.1
- `eslint-plugin-jsdoc` version: 61.1.12
